tldw_Server_API/app/core/DB_Management/ManuscriptDB.py

```
@@ -0,0 +1,824 @@
+                    entity="manuscript_scenes",
+                    entity_id=scene_id,
+                )
+
+            if row:
+                self._propagate_word_counts(conn, row["chapter_id"], row["project_id"])
+
+    # ------------------------------------------------------------------
+    # Word-count propagation
+    # ------------------------------------------------------------------
+
+    def _propagate_word_counts(self, conn: Any, chapter_id: str, project_id: str) -> None:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `conn`

(ANN401)